fe/be-java-extensions/java-udf/src/main/java/org/apache/doris/udf/UdfExecutor.java

```diff
@@ -503,4 +573,78 @@ protected void init(TJavaUdfExecutorCtorParams request, String jarPath, Type fun
+            switch (valueType) {
+                case BOOLEAN: {
+                    return new BuildMapFromType<keyType, Boolean>().get(keyCol, valueCol);
+                }
+                case TINYINT: {
+                    return new BuildMapFromType<keyType, Byte>().get(keyCol, valueCol);
+                }
+                case SMALLINT: {
+                    return new BuildMapFromType<keyType, Short>().get(keyCol, valueCol);
+                }
+                case INT: {
+                    return new BuildMapFromType<keyType, Integer>().get(keyCol, valueCol);
+                }
+                case BIGINT: {
+                    return new BuildMapFromType<keyType, Long>().get(keyCol, valueCol);
+                }
+                case LARGEINT: {
+                    return new BuildMapFromType<keyType, BigInteger>().get(keyCol, valueCol);
+                }
+                case FLOAT: {
+                    return new BuildMapFromType<keyType, Float>().get(keyCol, valueCol);
+                }
+                case DOUBLE: {
+                    return new BuildMapFromType<keyType, Double>().get(keyCol, valueCol);
+                }
+                case CHAR:
+                case VARCHAR:
+                case STRING: {
+                    return new BuildMapFromType<keyType, String>().get(keyCol, valueCol);
+                }
+                case DATEV2:
+                case DATE: {
+                    return new BuildMapFromType<keyType, LocalDate>().get(keyCol, valueCol);
+                }
+                case DATETIMEV2:
+                case DATETIME: {
+                    return new BuildMapFromType<keyType, LocalDateTime>().get(keyCol, valueCol);
+                }
+                case DECIMAL32:
+                case DECIMAL64:
+                case DECIMALV2:
+                case DECIMAL128: {
+                    return new BuildMapFromType<keyType, BigDecimal>().get(keyCol, valueCol);
+                }
+                default: {
+                    LOG.info("Not support: " + valueType);
+                    Preconditions.checkState(false, "Not support type " + valueType.toString());
+                    break;
+                }
+            }
+            return null;
+        }
+    }
+
+    public static class BuildMapFromType<T1, T2> {
+        public Object[] get(Object[] keyCol, Object[] valueCol) {
+            Object[] retHashMap = new HashMap[keyCol.length];
+            for (int colIdx = 0; colIdx < keyCol.length; colIdx++) {
+                HashMap<T1, T2> hashMap = new HashMap<>();
+                ArrayList<T1> keys = (ArrayList<T1>) (keyCol[colIdx]);
+                ArrayList<T2> values = (ArrayList<T2>) (valueCol[colIdx]);
+                for (int i = 0; i < keys.size(); i++) {
+                    T1 key = keys.get(i);
+                    T2 value = values.get(i);
+                    hashMap.put(key, value);
```

Comment:
keep the same as doris map if contain the key， just skip the put call continue of call put_if_not_exist